docs/spec/SPEC-SPEC.md

```diff
@@ -7,20 +7,30 @@ this directory.
 
 For consistency, specs should be written in passive present tense.
 
+## Pseudo-Code
+
+Pseudo-code should be avoided throughout the spec, instead bulleted-lists should 
```

Comment:
Hmmm, I don't 100% fully agree with this. I don't see why pseudo code can't be part of a spec. Imho, the pseudo code that exists in the vesting spec greatly improves the understanding and legibility of it. I agree that a spec should not be littered with pseudo code, but _clear and concise_ sections should be permitted.

Reply:
Agree with @alexanderbez here.

Reply:
I agree with the sentiment, I've updated this section to: 
```
Generally, pseudo-code should be minimized throughout the spec. Often, simple
bulleted-lists which describe a function's operations are sufficient and should
be considered preferable. In certain instances, due to the complex nature of
the functionality being described pseudo-code may the most suitable form of
specification. In these cases use of pseudo-code is permissible, but should be
presented in a concise manner, ideally restricted to only the complex
element as a part of a larger description. 
```

This said, I went through the vesting spec just now, and it's my opinion that the use of pseudo-code should be reduced... additionally, I'm not sure vesting as a "spec-within-a-spec" is the best, there is probably a better way to integrate vesting into auth